docs/internals/contributing/bugs-and-features.txt

```diff
@@ -126,45 +127,27 @@ benchmarks.
 How we make decisions
 =====================
 
-Whenever possible, we strive for a rough consensus. To that end, we'll often
-have informal votes on the Django Forum about a feature. In these votes we
-follow the voting style invented by Apache and used on Python itself, where
-votes are given as +1, +0, -0, or -1.
-Roughly translated, these votes mean:
-
-* +1: "I love the idea and I'm strongly committed to it."
-
-* +0: "Sounds OK to me."
-
-* -0: "I'm not thrilled, but I won't stand in the way."
-
-* -1: "I strongly disagree and would be very unhappy to see the idea turn
-  into reality."
-
-Although these votes are informal, they'll be taken very seriously. After a
-suitable voting period, if an obvious consensus arises we'll follow the votes.
-
-However, consensus is not always possible. If consensus cannot be reached, or
-if the discussion toward a consensus fizzles out without a concrete decision,
-the decision may be deferred to the :ref:`steering council <steering-council>`.
-
-Internally, the steering council will use the same voting mechanism. A
-proposition will be considered carried if:
-
-* There are at least three "+1" votes from members of the steering council.
-
-* There is no "-1" vote from any member of the steering council.
+Whenever possible, we strive for a rough consensus. To that end, we use emoji
+reactions to track the community response to a new feature idea.
+We give these meanings to these reactions:
 
-Votes should be submitted within a week.
+* 👍: This is something I would use
+* 👎: This is something that would cause problems for me or Django
+* 😕: I’m indifferent to this
+* 🎉: This is an easy win
 
-Since this process allows any steering council member to veto a proposal, a
-"-1" vote should be accompanied by an explanation of what it would take to
-convert that "-1" into at least a "+0".
+The Steering Council will regularly review the idea tracker and move ideas with
+community approval through the different stages of implementation:
 
-Votes on technical matters should be announced and held in public on the
-`Django Forum`_.
+* Idea
+* Approved - Idea refinement - Team creation
+* In progress
+* Working solution - Review - Feedback
+* Needs maintainer (Django only)
+* Done
```

Comment:
I think we need sort out the differences between the flow chart and the kanban board

Reply:
Yes, but maybe that can happen in a follow-up PR? I'd prefer to get this landed sooner than later.

Reply:
Alright, that's fair.